DiscordManimator.py

```diff
@@ -48,6 +48,13 @@ def construct(self):
 ```
 """)
 
+@bot.command()
+@commands.guild_only()
+async def m(ctx, *, arg): # alias for manimate
+    manimate(ctx, *, arg)
+    
+    
+
 @bot.command()
```

Comment:
Apparently you can just declare aliases

https://stackoverflow.com/a/50213130/6571126
https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Command.aliases

```suggestion
@bot.command(aliases=['m'])
```

Reply:
Thanks Eric, that is easy :)